pkg/artifactory/resource/configuration/resource_artifactory_ldap_group_setting.go

```diff
@@ -57,7 +57,7 @@ func ResourceArtifactoryLdapGroupSetting() *schema.Resource {
 			Type:             schema.TypeString,
 			Optional:         true,
 			Default:          "",
-			ValidateDiagFunc: validation.ToDiagFunc(utils.ValidateLdapDn),
+			ValidateDiagFunc: validation.ToDiagFunc(ValidateLdapDn),
```

Comment:
Can you do like you did with `validator.Cron` here?

Reply:
Sure thing.